cpu/cc26x0/periph/aux.c

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2016 Leon George
+ * (who won't go after you should you copy any part of this file ;-)
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+#include "cc26x0-aux.h"
+
+bool addi_reg_access_prepare(void)
```

Comment:
you are not using these functions -> remove them and add them when you need them later...